src/wanna/core/services/jobs.py

```diff
@@ -430,7 +433,7 @@ def write_manifest(
         Returns:
             Path: Path where resource manifest was saved to
         """
-
+        encryption_spec_key_name = self.config.gcp_profile.kms_key if self.config.gcp_profile.kms_key else None
```

Comment:
we should load the model `encryption_spec_key_name` and then fallback to `encryption_spec_key_name`